packages/plugin/src/rules/unique-fragment-name.ts

```diff
@@ -14,11 +14,7 @@ export const checkNode = (
   ruleName: string,
   messageId: string
 ): void => {
-  const documentName = node.name?.value;
-  if (!documentName) {
```

Comment:
I changed AST selector to `OperationDefinition[name!=undefined]` so this `if` no longer needed